src/main/scala/mesosphere/marathon/stream/EnrichedFlow.scala

```diff
@@ -0,0 +1,33 @@
+package mesosphere.marathon
+package stream
+
+import akka.NotUsed
+import akka.stream.scaladsl.{ Flow, Source }
+
+object EnrichedFlow {
+  @SuppressWarnings(Array("AsInstanceOf"))
+  def ignore[T]: Flow[T, Nothing, NotUsed] =
```

Comment:
It would be really nice to have some docs here.

Reply:
✅